torch/_meta_registrations.py

```diff
@@ -1636,3 +1629,60 @@ def upsample_nearest2d_vec(input, output_size, scale_factors):
 import torch._refs
 import torch._refs.nn.functional
 import torch._refs.special
+
+
+def activate_meta():
+
+    activate_meta_table = {}
+
+    for type in ["meta", "post_autograd", "pre_autograd"]:
+        registry = global_decomposition_table[type]
+
+        for opo in registry:
+            if opo not in activate_meta_table:
+                activate_meta_table[opo] = registry[opo]
+
+    for op_overload, fn in activate_meta_table.items():
+        assert isinstance(op_overload, OpOverload)
+
+        op_overload.py_impl(torch._C.DispatchKey.Meta)(fn)
+
+        dispatch_key_registration = torch._C._dispatch_dump(op_overload.name())
```

Comment:
don't do it this way; use one of the more specific dispatch query functions

Reply:
Shall I also check for CompositeImplicitAutogradNestedTensor? 

aten.unbind has this Dispatchkey.

Reply:
I suppose it's not needed? 

For non-nested tensor, aten.unbind should still depends on python's meta.

Reply:
uhhh... library.py is checking CompositeImplicitAutogradNestedTensor


I am fix this in the next PR.

Reply:
for the CompositeImplicitAutograd test you just check if the (non-computed) key is registered